Panic when exec completion without args

# Expected Behavior

Not sure, but it shouldn't print outputs:

```bash
$ tkn completion
$
```

# Actual Behavior

```bash
$ tkn completion 
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/tektoncd/cli/pkg/cmd/completion.Command.func1(0xc0000e6f00, 0x25c5078, 0x0, 0x0, 0x0, 0x0)
	/workspace/src/github.com/tektoncd/cli/pkg/cmd/completion/completion.go:50 +0x15d
github.com/tektoncd/cli/vendor/github.com/spf13/cobra.(*Command).execute(0xc0000e6f00, 0x25c5078, 0x0, 0x0, 0xc0000e6f00, 0x25c5078)
	/workspace/src/github.com/tektoncd/cli/vendor/github.com/spf13/cobra/command.go:826 +0x465
github.com/tektoncd/cli/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc0000e6c80, 0xc0000a8e70, 0xc0000e6c80, 0xc000048118)
	/workspace/src/github.com/tektoncd/cli/vendor/github.com/spf13/cobra/command.go:914 +0x2fc
github.com/tektoncd/cli/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	/workspace/src/github.com/tektoncd/cli/vendor/github.com/spf13/cobra/command.go:864
main.main()
	/workspace/src/github.com/tektoncd/cli/cmd/tkn/main.go:31 +0x4c
```

# Steps to Reproduce the Problem

1. Download and build the latest release (using `go get`)
2. Run `tkn completion`

--- a/pkg/cmd/completion/completion.go
+++ b/pkg/cmd/completion/completion.go
@@ -50,11 +50,13 @@ func Command() *cobra.Command {
 			"commandType": "utility",
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			switch args[0] {
-			case "bash":
-				return cmd.Root().GenBashCompletion(os.Stdout)
-			case "zsh":
-				return runCompletionZsh(os.Stdout, cmd.Parent())
+			if len(args) == 1 {
+				switch args[0] {
+				case "bash":
+					return cmd.Root().GenBashCompletion(os.Stdout)
+				case "zsh":
+					return runCompletionZsh(os.Stdout, cmd.Parent())
+				}
 			}
 			return nil
 		},

